### ISSUE TITLE
TS plugin doesn't fully work if using workspace typescript version

### ISSUE THREAD

**Issue by ananni13** [johnsoncodehk/volar#169]
If I set VSCode to use the workspace version of typescript instead of its own (`> TypeScript: Select TypeScript version...` -> `Use Workspace Version`) and remove the `*.d.ts` file, it shows this error even with Volar TS plugin enabled:

![ts_error](https://user-images.githubusercontent.com/37385837/116934048-b814a580-ac64-11eb-85ce-e2b3ea908c47.jpg)
`Cannot find module './App.vue' or its corresponding type declarations. ts(2307)`

The error goes away if I set `> TypeScript: Select TypeScript version...` -> `Use VS Code's Version`

Is this intended?

If you need a repo to test it, I created a clean one here (it's just the @vitejs/create-app vue-ts preset) with the `shims-vue.d.ts` file commented out: https://github.com/ananni13/volar-ts-test

**Comment by johnsoncodehk** [comment]
Thanks for your repro case!

VSCode using built-in TS library by default, and VSCode provide `typescriptServerPlugins` contribute point to allow VSCode extension setup a TS plugin for built-in TS library.

VSCode extension can't setup a TS plugin for workspace TS library, so when you using workspace TS version, you need to config the [Volar TS plugin](https://github.com/johnsoncodehk/volar/tree/master/packages/typescript-vue-plugin) to workspace tsconfig.json [plugins](https://www.typescriptlang.org/tsconfig#plugins) option.

But I have not released Volar TS plugin to npm yet. 😅

**Comment by productfrontenddeveloper** [comment]
> Published in 0.25.3: https://www.npmjs.com/package/typescript-vue-plugin

tsconfig.json 

```
 "plugins": [
      {
        "name": "typescript-eslint-language-service"
      },
      {
        "name": "typescript-eslint-language-service"
      }
    ],
    ```
    added or no add ? which one should i do
